cluster/vagrant/setup_kubernetes_master.sh

```diff
@@ -50,6 +50,31 @@ else
   kubectl create -f kube-$NETWORK_PROVIDER.yaml
 fi
 
+{
+  pushd /vagrant
+  source hack/config.sh
+  popd
+
+  for SVC in spice-proxy:3128 virt-controller:8182 virt-api:8182 haproxy:8184 virt-manifest:8186;
+  do
```

Comment:
I'd rather see this as part of 'make vagrant-deploy' than being performed during the initial 'vagrant up'. 

My concern is that changes to ingress shouldn't require someone to rebuild vagrant.

Reply:
So ... We can do it there as well, obviously.

One of the things I'm trying to achieve is a spearation of the internal setup from the external access.
With this patch set this is achieved, because `manifests/` only contains internal pod definitions.
And ingress (even if not an ingress object) is left to the env setup - that's why I moved it to the vagrant setup.

In this setup, I can take the `manifests/` directory and use it for minikube, and adding a note that the user needs to run `kubectl expose service spice-proxy …` to get spice/console access.
What I want to say: The `manifest/` manifests are sufficient and independent to define KubeVirt in the cluster.

By moving the external services into `manifests/` we are pulling back env dependent configs into the manifest dir.

Reply:
@fabiand how about moving the ingress service generation code to cluster/deploy.sh then?

Reply:
Great idea!